tensorflow/compiler/xla/service/gpu/cudnn_conv_runner.cc

```diff
@@ -188,9 +189,11 @@ StatusOr<CudnnConvParams> GetCudnnConvParams(
   const Shape* filter_shape;
   const Shape* output_shape;
 
-  params.algorithm = se::dnn::AlgorithmConfig(se::dnn::AlgorithmDesc(
-      backend_config.algorithm(), backend_config.tensor_ops_enabled()),
-      backend_config.scratch_size());
+  // The third field is scratch size stored from conv_algorithm_picker
+  params.algorithm = se::dnn::AlgorithmConfig(
+      se::dnn::AlgorithmDesc(backend_config.algorithm(),
+                             backend_config.tensor_ops_enabled()),
+      conv->shape().tuple_shapes(1).dimensions(0));
```

Comment:
@jerryyin it's non-trivial to deduce `conv->shape().tuple_shapes(1).dimensions(0)` contains `scratch_size`, could you revise the comment above to provide some pointers to the relevant source code?

Reply:
Will add more comment comment to address this in the code. 

To give you a pointer here, the scratch_bytes was added as convolution instructions' shape in this [line](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/gpu/cudnn_conv_algorithm_picker.cc#L537)